src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -630,42 +632,6 @@ def process_scores_(
         # TODO: should we give num_entities in the constructor instead of inferring it every time ranks are processed?
         self.num_entities = num_entities
 
-    def evaluate_ogb(
```

Comment:
@mberr are there any issues with removing this now-duplicate code?

Reply:
no, I don't think so.